### ISSUE TITLE
Unable to open asset URL: file:///android_asset/www/+++/index.html

### ISSUE THREAD

**Issue by AKarthikeyan1982** [tkyaji/cordova-plugin-crypt-file#55]
Hi Team,
I have tried to use the cordova-plugin-crypt-file for my hybrid app and its showing the 'E/AndroidProtocolHandler(2434): Unable to open asset URL: file:///android_asset/www/+++/index.html' error in logCat. Please help.
![capture](https://user-images.githubusercontent.com/7930681/29004293-9fae9594-7ae2-11e7-92b8-a6b4f7b002fd.PNG)

**Comment by Medtrack** [comment]
Hello, 
I can confirm this issue.

**Comment by renandecarlo** [comment]
But does the plugin work? It is meant to throw this error...

**Comment by AKarthikeyan1982** [comment]
Yes, the plugin is working with the reported warning... Is that any way to
fix the issue?

**Comment by ealch** [comment]
![assets-crypt-error](https://user-images.githubusercontent.com/7937529/35801423-a530cc48-0a6c-11e8-8587-125fd64539a6.png)
I can confirm it too...  Do you have any news?

**Comment by SleimanB** [comment]
Any news about this issue?

**Comment by SleimanB** [comment]
I don't know if you are using Ionic 3 Framework but for the issue was ` cordova-plugin-ionic-webview ` so when I removed it my problem was solved @AKarthikeyan1982 @Medtrack @ealch

**Comment by AKarthikeyan1982** [comment]
Hi,
Thanks you for the suggestion, am not using ionic, am using onsenui and
it's still having the issue.

Regards,
Karthikeyan A.

**Comment by SleimanB** [comment]
I Suggest removing your plugins one by one and see if the problem will be resolved. because I think the problem is from one of your plugins @AKarthikeyan1982

**Comment by ZubeeDoo** [comment]
My app consists of cordova-plugin-whitelist and cordova-plugin-crypt-file only. Still the issue...

**Comment by ZubeeDoo** [comment]
Solved.
OK ,  Exactly not sure on what i did but now it works (I am using angular and angular material with cordova).
Walk through : 
 1. unzip your cordova built (just rename from .apk to .zip and extract like always) and find your index.html file path it will be inside www within www(created by the plugin). just make sure where your files are. Then open plugins/cordova-plugin-crypt-file/hooks/after_prepare.js and change line 65 from  child.attrib.src = '/+++/' + child.attrib.src;  to child.attrib.src = '/www/' + child.attrib.src; 
                  --- it worked partially for me but not the correct solution
2.simple reinstall the plugin.
-- if its still not working then Number 3. create a new cordova project (just a blank one without any additional plugins ) and then install the plugin.
I hope this helps someone. Sorry, if it doesn't.

**Comment by A110SHT** [comment]
Hello everyone after resolve the **file:///android_asset/www/+++/index.html**  

Although it successfully encrypts all the js and HTML files, after installing the apk it shows whole encrypted string. So the device doesn't decrypt. What could be the reason?
I see the random string in an app. It will don't decrypt as expected.
how to resolve this?

**Comment by A110SHT** [comment]
this plugin is not working on ionic project how to make work in ionic.

**Comment by devitito** [comment]
+1 Same problem on android with cordova-plugin-ionic-webview
